ptvsd/wrapper.py

```diff
@@ -1914,6 +1962,18 @@ def on_pydevd_thread_suspend(self, seq, args):
             pydevd_comm.CMD_ADD_EXCEPTION_BREAK
         }
 
+        xframes = list(xml.thread.frame)
+        xframe = xframes[0]
+        filepath = unquote(xframe['file'])
+        if reason in STEP_REASONS or reason in EXCEPTION_REASONS:
+            if not self._should_debug(filepath):
+                self.pydevd_notify(pydevd_comm.CMD_THREAD_RUN, pyd_tid)
+                return
+            for filename in IMPORTLIB_BOOTSTRAP:
+                if filepath.endswith(filename):
+                    self.pydevd_notify(pydevd_comm.CMD_THREAD_RUN, pyd_tid)
```

Comment:
This should not be needed after I integrate the latest changes from pydevd to handle (see: https://github.com/fabioz/PyDev.Debugger/blob/master/_pydevd_bundle/pydevd_frame_utils.py#L44)